djstripe/models/base.py

```diff
@@ -156,6 +156,11 @@ def get_stripe_dashboard_url(self) -> str:
             item = self.stripe_dashboard_item_name
             return f"{base_url}{item}/{self.id}"
 
+    @staticmethod
+    def extract_stripe_account(data):
```

Comment:
Just make this a .stripe_account property

Reply:
What is the advantage of doing that? And I do not understand how will this be converted into a property. We will unnecessarily pass `self` and track state across multiple instances when it is only a function of the `data` dictionary. Or am I misunderstanding you?

Reply:
I misread the code... Sorry, I don't see how this is a useful method. I think it's an unnecessary additional abstraction layer that will never be overridden.

Reply:
@jleclanche The intention was not to make it overridable but to provide a uniform interface to extract stripe accounts across various methods.

Reply:
If it's a static method and not meant to be overridable then it doesn't really make sense to me. Just get rid of it please. Put it in utils.py if you really want to DRY this, but IMO it's extremely superfluous.

Reply:
@jleclanche This is fixed. Please review again.